modules/react/toast/lib/Toast.tsx

```diff
@@ -93,9 +94,10 @@ export const Toast = createComponent('div')({
         width={toastWidth}
         padding="s"
         depth={5}
-        role={isInteractive ? 'dialog' : 'status'}
-        aria-live={isInteractive ? 'off' : 'polite'}
+        role={isInteractive ? 'dialog' : isError ? 'alert' : 'status'}
+        aria-live={isInteractive ? 'off' : isError ? 'assertive' : 'polite'}
         aria-atomic={!isInteractive}
+        tabIndex={1}
```

Comment:
Q: I don't see this `tabIndex` listed in [this issue](https://github.com/Workday/canvas-kit/issues/429). Can you give me more context on why it's added here. please?

Reply:
Ha, it was just for testing! I forgot to remove it.